_data/versions.yml

```diff
@@ -15,6 +15,8 @@ master:
      calico/cni:
       version: master
       url: ""
+      download_calico_url: https://github.com/projectcalico/cni-plugin/releases/download/v1.6.0/calico
```

Comment:
This will need sorting out properly to reference the builds on CircleCI - but that will require an update to the projectcalico site - suggest doing that as a separate PR.